.gitignore

```diff
@@ -42,6 +43,8 @@ config/application.yml
 tmp/
 data/
 log/*
+.cache
+.local
```

Comment:
I'm also getting a `.bash_history` in the main directory after doing a `make bash`, could be ignored here.

Reply:
good catch, done!